src/lib/mdns/ServiceNaming.cpp

```diff
@@ -172,5 +173,24 @@ CHIP_ERROR MakeServiceSubtype(char * buffer, size_t bufferLen, DiscoveryFilter s
     return (requiredSize <= (bufferLen - 1)) ? CHIP_NO_ERROR : CHIP_ERROR_NO_MEMORY;
 }
 
+CHIP_ERROR MakeCommissionableNodeServiceTypeName(char * buffer, size_t bufferLen, DiscoveryFilter nameDesc)
+{
+    char subtypeQualifier[] = "_sub";
```

Comment:
Already defined in `kSubtypeServiceNamePart` and `kCommissionableServiceName`

Reply:
Thanks - changed.